docs/worldgen/biomemodifier.md

```diff
@@ -0,0 +1,648 @@
+# Biome Modifiers
+
+Biome Modifiers are a data-driven system that allows for changing many aspects of a biome. Ranging from injecting or removing PlacedFeatures, adding or deleting mob spawns, changing the climate, and adjusting foliage and water color. NeoForge provides several default Biome Modifiers that covers the majority of use cases for both players and modders.
+
+For players and pack makers, you will want to see the [#Applying-Biome-Modifiers](#Applying-Biome-Modifiers) section and [#Builtin-Neoforge-Biome-Modifiers](#Builtin-Neoforge-Biome-Modifiers) section.
+
+For modders looking to do basic goals, you will want to see the [#Applying-Biome-Modifiers](#Applying-Biome-Modifiers) section, [#Builtin-Neoforge-Biome-Modifiers](#Builtin-Neoforge-Biome-Modifiers) section, and possibly the [#Datagenning-Biome-Modifiers](#Datagenning-Biome-Modifiers) section if you prefer to datagen.
+
+For modders looking to do advance custom Biome Modifiers, check out the [#Applying-Biome-Modifiers](#Applying-Biome-Modifiers) section and [#Creating-Custom-Biome-Modifiers](#Creating-Custom-Biome-Modifiers) section.
```

Comment:
I kind of view it like a table of contents that leads you down the right path. But a separate file for built it could work and then have a `For Players and Pack Makers` section here that says what they need to know and links to the other page for them to get started with making the JSON.

An then another section of `For Modders` that tells the modders what they need to know and links to the builtin page if they want to handwrite json and links to datagen section if they prefer datagen and links to custom section too.

Reply:
Yeah, I understand the purpose of the quantifiers of who should read what, it just seems a bit like you only need to read section X and Z to do what you need. Generally, the flow of the document should notify readers that they no longer need to go any further.

Trying to think of a way to keep it, I think it would make more sense as a list or table that's like 'Relevant Sections for' and then having the links as a list rather than a paragraph.